content/en/synthetics/private_locations/_index.md

```diff
@@ -138,6 +138,8 @@ Run this command to boot your private location worker by mounting your configura
 docker run --rm -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetics-check-runner.json datadog/synthetics-private-location-worker:latest
 ```
 
+Ensure that youre `<MY_WORKER_CONFIG_FILE_NAME>.json` file is not in the root home folder (`/root`), but in `/etc/docker`.
```

Comment:
```suggestion
Ensure that your `<MY_WORKER_CONFIG_FILE_NAME>.json` file is in `/etc/docker`, not the root home folder.
```
I took out the `/root` part because it's too easy to skim and misunderstand. Also, I think you should move this info _before_ the shell command, so they check before they run it.